internal/incident/incident.go

```diff
@@ -86,13 +85,12 @@ func (i *Incident) ProcessEvent(ev event.Event, created bool) error {
 		if err != nil {
 			return err
 		}
+
+		i.logger = i.logger.With(zap.String("object", i.ObjectDisplayName()), zap.String("incident", i.String()))
```

Comment:
You shouldn't use `.With()` to update existing fields, they just get accumulated (note the duplicate `object`):

```
2023-05-26T11:27:00.097+0200    INFO    incident        Incident severity changed from ok to warning    {"object": "dc1.example.com: memory", "object": "dc1.example.com: memory", "incident": "#3809"}
```